キャラクターの追加

Mamonbo opened the pull request:
acd8ebf4e1a43bf4fd4ad76d1e7621da692b1c3e
コンフリクトは解消しておいたのでご安心を

Mamonbo committed:
Merge pull request #2 from l1048576/master

時代に合わせよう

Mamonbo committed:
Merge pull request #3 from l1048576/master

時代に合わせよう

Mamonbo committed:
人物の追加

世界観クラッシャーなので二次創作でもいいです。

Mamonbo committed:
Merge branch 'master' of github.com:Mamonbo

コンフリクトの解消
なんとかなったかな

Mamonbo committed:
文のフォーマットの修正

lo48576 commented on settings/characters.adoc:
```diff
@@ -53,3 +53,14 @@
+- 前過程の人なので男性な可能性もある。
+- この人のためだけに英語第八 第九 が開かれている。
+- 元ネタはクォーター制になって英語の科目がクォーターごとに分かれたとこ
+  ろから。
```
日本語は分かち書きじゃないので、こういうwrapは無しで

l1048576 commented:
ここに来て急にシリアス風味出してきたな……ww

Mamonbo committed:
改行位置の修正

> 入力していって行を長くしすぎた場合、自動的に単語と単語の間で行が分割さ
れます（日本語の場合はどの文字の間でも。禁則処理もなされます）。

emacs入門ガイドより

第1話感を出してみる。
~~生のファイル読む人いるのかな~~

l1048576 commented:
改行は消えても半角空白は全部は消えないから、変換後のhtmlとかでも日本語文の変なところにスペース空くし、原則日本語文は句読点か一部記号の直後以外で改行すべきじゃないんだよねぇ……(まあ日本語の分かち書きじゃないのが少数派なだけかもしれないが)

それはともかく問題なさそうなのでmergeします
